### DISCUSSION TITLE
application not able to delete from UI .It got stuck

### DISCUSSION THREAD

**Discussion by Srivalle** [argoproj/argo-cd#12492] [Q&A]
I am not able to delete the application from argocd ui and also from cli. 
I am getting the following error 
FATA[0010] rpc error: code = InvalidArgument desc = error patching application with finalizers: Application.argoproj.io "XXXX" is invalid: metadata.finalizers: Forbidden: no new finalizers can be added if the object is being deleted, found new finalizers []string{"resources-finalizer.argocd.argoproj.io"} . 
I am using 2.5.7 version of arogcd . Anyone could you please help here .

**Comment by aikijitsu** [comment]
In my case I had to go to the azure portal and delete the pod running, then it disappeared from the ui in argocd.
